databroker/mongo_normalized.py

```diff
@@ -440,6 +465,21 @@ def metadata(self):
             ]
         metadata = dict(collections.ChainMap(transformed, self._metadata))
         return metadata
+    
+    async def update_metadata(self, metadata=None, specs=None):
```

Comment:
This one should be working with `metadata["descriptors"]` and, correspondingly:

```py
for descriptor in metadata["descriptors"]:
    self.serializer.update("descriptor", descriptor)
```

A prerequisite is the add support for updating `"descriptor"` documents in `suitcase.mongo_normalized`.

Reply:
Support added in 63ed99bffc409add78f86afcfb66318f66216acd
Changes just need to be merged in `suitcase-mongo` and then CI can be run again to pass all tests related to it.